gcd/src/main/java/org/spine3/sample/gae/Application.java

```diff
@@ -55,8 +55,8 @@ public static void main(String[] args) {
         final LocalDatastoreStorageFactory storageFactory = LocalDatastoreStorageFactory.getDefaultInstance();
         storageFactory.getOptions().setEventIteratorPageSize(ITERATOR_PAGE_SIZE);
```

Comment:
As I see it, we provide `DatastoreStorageFactory.Options#getEventIteratorPageSize` as part of the API. I'll add a clarifying comment instead of removing.